gallery_dl/extractor/tiktok.py

```diff
@@ -143,12 +144,26 @@ def _generate_avatar(self, avatar_url, data, user_name, user_id):
     def _sanitize_url(self, url):
         return text.ensure_http_scheme(url.replace("/photo/", "/video/", 1))
 
-    def _extract_rehydration_data(self, url):
-        html = self.request(url).text
-        data = text.extr(
-            html, '<script id="__UNIVERSAL_DATA_FOR_REHYDRATION__" '
-            'type="application/json">', '</script>')
-        return util.json_loads(data)["__DEFAULT_SCOPE__"]
+    def _extract_rehydration_data(self, url, *, retries=None):
+        if retries is None:
+            retries = self._retries
+        try:
+            html = self.request(url).text
+            data = text.extr(
+                html, '<script id="__UNIVERSAL_DATA_FOR_REHYDRATION__" '
+                'type="application/json">', '</script>')
+            return util.json_loads(data)["__DEFAULT_SCOPE__"]
+        except JSONDecodeError:
```

Comment:
```suggestion
        except ValueError:
```
Avoids having to import `JSONDecodeError`